lib/widgets/editor/tree_widgets/tree_card_node.dart

```diff
@@ -2,6 +2,7 @@ import 'package:flutter/material.dart';
 
 class TreeCardNode extends StatelessWidget {
   final Widget title;
+  final Widget? icon;
```

Comment:
Should probably be renamed to `leading` to match other flutter widget conventions since this doesn't strictly have to be an icon.

Reply:
Fair point, updated it to `leading` and all of its references